### PR TITLE
{chem}[foss/2017b,intel/2017b] charmm v43b2

### PR DESCRIPTION
**Pull request by vanzod**
The `intel/2017b` build requires the following changes in the easyblock:
~~https://github.com/easybuilders/easybuild-easyblocks/pull/1707~~

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/c94c67d4ecfd187fbf956f0bd39da101 for a full test report.

File: easybuild/easyconfigs/c/charmm/charmm-43b2-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+easyblock = 'EB_CHARMM'
+
+name = 'charmm'
```
Comment (boegel) [inline easybuild/easyconfigs/c/charmm/charmm-43b2-foss-2017b.eb]: @vanzod Why `charmm` rather than `CHARMM` (cfr. description)?

Comment (vanzod) [reply easybuild/easyconfigs/c/charmm/charmm-43b2-foss-2017b.eb]: `charmm` is the non-free version of `CHARMM`. I have added a line in the description to highlight the difference.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/29f46f598fa2ee198f0250765b6cd2b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2543.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/58f78b247dda01193f1eaba1b555024b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8bda0bf1bcf648a3eda13b031b9cdb98 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**